### ISSUE TITLE
[Suggestion/Question]

### ISSUE THREAD

**Issue by Colawaver** [ejektaflex/Bountiful#133]
I've tried to look around for a way to integrate the bounty board %, decrees and rewards onto **JEI**, But i've come up almost empty handed.
What are the chances this exist or can be implemented?

Currently building a fairly decent sized modpack **_gated and progression locked behind rewards from the bounty board_** and this would be helpful.
And yes, I can add tooltips/descriptions to each of the items which is a hassle but do-able.
with a hassle i mean, if I take an item and gate it behind the board and remove the recipe it wont show up 'where'
(And not everyone read descriptions and gives up easily)

Or maybe a option to turn it on and off?
Since I can see how it would be a lot of clutter on JEI with all the default decrees which one cant turn off. (yet(see someone suggested that))

**Comment by ejektaflex** [comment]
Currently, locking progression can only be done via the 1.12, which has Game Stages support. It would be nice to add in support for that here, but I haven't had time. If given the time, I'd love to add some further JEI support :^) Otherwise, the closest I have is the aforementioned compat in the 1.12 version, but I frankly wouldn't suggest it when the 1.14-1.16 versions are so much nicer.

**Comment by Colawaver** [comment]
I see, unfortunately I'm already well on my way in 1.16.5.
Currently the progression has been locked behind items custom and other items which their recipes and processing has been removed or altered to fit the progression wanted. 
The issue I've ran into is when the player wants to see how they can unlock or progress, the item(s) that has been altered to only be available in bounty board doesnt show up in JEI when searching.

This can be of course be worked around with JEI support, adding descriptions to all the items or a questbook explaining.
Questbook would personally be out of the question as of how I've built the pack till now.

there wouldn't be a way to communicate with you in a form of private dms?
